.github/scripts/trymerge.py

```diff
@@ -718,9 +724,26 @@ def merge_ghstack_into(self, repo: GitRepo, force: bool, comment_id: Optional[in
             msg += f"\nApproved by: {approved_by_urls}\n"
             repo.amend_commit_message(msg)
 
-    def merge_into(self, repo: GitRepo, *, force: bool = False, dry_run: bool = False, comment_id: Optional[int] = None) -> None:
+    def merge_into(self, repo: GitRepo, *,
+                   force: bool = False,
+                   dry_run: bool = False,
+                   already_merged: bool = False,
+                   comment_id: Optional[int] = None) -> None:
         # Raises exception if matching rule is not found
         find_matching_merge_rule(self, repo, force=force, skip_internal_checks=can_skip_internal_checks(self, comment_id))
+        if not already_merged:
```

Comment:
This is slightly confusing--is this just a "merge only the first time" type of thing? The terminology of already merged is ambiguous, it sounds like it should be more like..already squashed, or something like that.